api/task.go

```diff
@@ -142,6 +144,14 @@ type TaskDatabaseRestorePayload struct {
 	BackupID     int    `json:"backupId,omitempty"`
 }
 
+// TaskDatabasePITRPayload is the task payload for database PITR restore.
+type TaskDatabasePITRPayload struct {
```

Comment:
Do we need to split a PITR stage into more than one tasks? E.g. sync -> cut over -> drop shadow db.